core/templates/dev/head/components/side_navigation_bar/side_navigation_bar_directive.html

```diff
@@ -77,6 +77,9 @@
       height: 56px;
       padding-top: 6px;
     }
+    .oppia-sidebar-menu-icon.fa {
```

Comment:
Sure, will address the first comment in my next push. Coming to the second point, as stated in the Note section in PR's description, the get started icon (rocket) is not available in material icons. It is currently exclusive icon of fontawesome. Hence, fa class was added.